static/css/legacy.less

```diff
@@ -2669,9 +2669,9 @@ ul {
 }
 
 #panelsz {
```

Comment:
#panelsz is not a valid selector.

Reply:
??

Reply:
Sorry the confusion, I got confused with the ".panels" class and thought this was a typo. I see now you added the id as part of this change. We should using meaningful names. "panelsz" is not a word in the dictionary.

So a bit of confusion on my side, but issue with this patch still stands.
For the same reasons as I've mentioned on the other patch, we should not be using ID selectors to fix these kind of problems. ID selectors create lots of problems in projects with complicated CSS like this one. I have been working to remove them in the codebase so adding more makes that cleanup even more difficult (a patch is in flight that will mean changes like this fail linting). http://oli.jp/2011/ids/ has a lot of strong arguments about why ID selectors are problematic. ID selectors should be added for JavaScript and to markup accessible fragments of the document, not for styling.

While it fixes the issue you've rightly identified, it does so in such a way that's not sustainable and fixes it in such a way that's not obvious. If we find we want to style ".panels" differently on another page, we'll have to make the CSS rule more powerful than this one e.g. #panelsz.panels - that's something I'm keen to avoid.

You could also fix the problem like so:
```
/*
This CSS selector is overly specific so that it can override the .Tools .panel
width style for the work pages.
See more information @ https://github.com/internetarchive/openlibrary/issues/1228
*/
.workDetails .Tools .panel {
    width: auto;
}
```

What's more interesting to me is do we need this rule at all:
```
.Tools .panel {
    width: 250px;
    margin-bottom: 10px;
}
```
Can it be removed? Or maybe changed to look like so:
```
.Tools .panel {
    width: auto;
    margin-bottom: 10px;
}
```
This change may cause a regression elsewhere, if width 250px is needed in some cases, so it's important to spend some time investigating the different options.

Reply:
well, `panelsz` was meant as an abbreviation of `panelsize` but ok i see your points.
I'll update later after more testing.

Reply:
good catch! thanks @jdlrobson

Reply:
I removed `width` property after I checked for possible regressions. That also fixed the main issue which the initial PR addresses. `margin-bottom` might also be ok to be removed but I left it since there is a slight effect.